### PR TITLE
CVE-2014-0050

### PR DESCRIPTION
**Pull request by Anorwen**
Comparing Tomcat CVE YAML.

### CODE REVIEW

File: cves/CVE-2014-0050.yml [hunk 1]
```diff
@@ -4,33 +4,33 @@ nickname_instructions: |
...
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: Multipart Infinite Loop
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 400
```
Comment (TheGuywithTheHat) [inline cves/CVE-2014-0050.yml]: I feel like CWE-400 is more about the _consequences_ of the vulnerability. CWE-20 might be more descriptive of the vulnerability itself.

File: cves/CVE-2014-0050.yml [hunk 2]
```diff
@@ -43,7 +43,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: A software package offered by Apache Tomcat for hosting websites 
+  allowed attackers to consume near 100% of the host's CPU, creating a Denial of
+  Service attack. This exploit involved an attacker tinkering with the 
+  Content-Type header when uploading a file to the server, preventing the system
+  from exiting an infinite loop.
```
Comment (TheGuywithTheHat) [inline cves/CVE-2014-0050.yml]: It might be helpful to add a description of what exactly triggers the exploit.

File: cves/CVE-2014-0050.yml [hunk 3]
```diff
@@ -101,9 +101,14 @@ unit_tested:
...
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: The edge cases for this vulnerability were not tested for. Upon inspection
+    of the code they do have variables for the boundaries, however they are not handled
+    properly or they are never used.
+
+    When this issue was fixxed they made use of the variables that were present. They
```
Comment (TheGuywithTheHat) [inline cves/CVE-2014-0050.yml]:
```suggestion
    When this issue was fixed they made use of the variables that were present. They
```

File: cves/CVE-2014-0050.yml (hunk 1 above)
Comment (Anorwen) [reply cves/CVE-2014-0050.yml]: After looking at your CVE I agree with you. I'll change this one it to CWE-20.

File: cves/CVE-2014-0050.yml (hunk 2 above)
Comment (Anorwen) [reply cves/CVE-2014-0050.yml]: I'll add something about that: it was mostly due to changing the boundaries in the request.
